### PR TITLE
wmctrl: add page

### PR DESCRIPTION
**Pull request by slashleo**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages/linux/wmctrl.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+- Switch to the first window whose (partial) title matches:
+
+`wmctrl -a {{window_title}}`
+
+- Move a window to the current workspace, raise it and give it focus:
+
+`wmctrl -R {{window_title}}`
+
+- Switch to workspace:
```
Comment (zdroid) [inline pages/linux/wmctrl.md:17]:
```suggestion
- Switch to a workspace:
```

File: pages/linux/wmctrl.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+`wmctrl -s {{workspace_number}}`
+
+- Select a window and toggle fullscreen:
+
+`wmctrl -r {{window_title}} -b toggle,fullscreen`
+
+- Select a window a move it to workspace:
```
Comment (zdroid) [inline pages/linux/wmctrl.md:25]:
```suggestion
- Select a window a move it to a workspace:
```

**Comment by zdroid** [comment]
Just two minor suggestions from me.

File: pages/linux/wmctrl.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# wmctrl
+
+> Command line interaction with X Window Manager.
```
Comment (andrik) [inline pages/linux/wmctrl.md:3]:
```suggestion
> CLI for X Window Manager.
```

**Comment by andrik** [comment]
@slashleo I noticed it was your first contribution to tldr project

Thank you one more time

**Comment by slashleo** [comment]
Thank you too :)

**Pull request merged**